packages/atti-components/src/components/Stepper/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+import Step from './styles/StyledStep'
+import StyledLabel from './styles/StyledLabel'
+import { StepperProvider } from './hooks'
+
+/**
+ * A basic stepper
+ */
+
+const Stepper = ({ variant, children }) => {
+  return <StepperProvider variant={variant}>{children}</StepperProvider>
+}
+
+Stepper.defaultProps = {
+  variant: 'number',
+}
+
+Stepper.propTypes = {
+  /**
+   * The variant of the stepper.
+   */
+  variant: PropTypes.oneOf(['number', 'checks', 'normal']),
+  children: PropTypes.any.isRequired,
+}
+
+Stepper.Step = Step
+Stepper.Label = StyledLabel
+
+/** @component */
+export default styled(Stepper)``
```

Comment:
The DOM / accessibility of your component can be improved. You can find some tips here  https://stackoverflow.com/a/52935539
If you need help, don't hesitate to come to me.